manifests/t/TrainerRoad/TrainerRoad/2023.1.2.166/TrainerRoad.TrainerRoad.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: TrainerRoad.TrainerRoad
+PackageVersion: 2023.1.2.166
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://trainrdtrcmn01un1softw01.blob.core.windows.net/installers/windows/v001/Production/TrainerRoad.exe
+  InstallerSha256: 752F97E089D3CF31A85EE4F1E1F7F9B51E676FA18A72405DB7218B5B20709442
```

Comment:
```suggestion
  InstallerSha256: A01164CD6BB9C936E5CDBA9FF762612D6B267977B5C431D21979A20F19E74B58
```

Reply:
Hi @KevinLaMS,

Apparently, the developer has modified the package. Possibly with a bugfix.